ts/src/btcalpha.ts

```diff
@@ -398,7 +398,7 @@ export default class btcalpha extends Exchange {
         const marketId = this.safeString (trade, 'pair');
         market = this.safeMarket (marketId, market, '_');
         const timestampRaw = this.safeString (trade, 'timestamp');
-        const timestamp = this.parseNumber (Precise.stringMul (timestampRaw, '1000000'));
+        const timestamp = this.parseToInt (Precise.stringMul (timestampRaw, '1000000'));
```

Comment:
@ttodua curiously, I already have these fixes on the c# branch 😅 
![image](https://user-images.githubusercontent.com/43336371/236944180-4566dba8-948f-44ed-a193-ef782a2808d0.png)

Reply:
maybe we can merge separated PR, so those other branches (your c# and my tests PR) would be cleaner

Reply:
btw, as I see, we don't need to wait travis build passed for this PR atm, as the failed exchanges are others , with temporary outages.